services/api/service.go

```diff
@@ -1017,6 +1041,18 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		return
 	}
 
+	epoch := payload.Slot() / uint64(common.SlotsPerEpoch)
```

Comment:
whether it's capella needs to be detected by api.headSlot, rather than the builder submissions. let me add a commit here!